python/pythonizations/SmartPtr.C

```diff
@@ -1,18 +1,18 @@
 #include "SmartPtr.h"
 
-auto_ptr<MyShareable> gime_mine() { return mine; }
-auto_ptr<MyShareable>* gime_mine_ptr() { return &mine; }
-auto_ptr<MyShareable>& gime_mine_ref() { return mine; }
+shared_ptr<MyShareable> gime_mine() { return mine; }
+shared_ptr<MyShareable>* gime_mine_ptr() { return &mine; }
```

Comment:
As far as I can tell such behaviour is not relevant is this test (i.e. any smart pointer would do fine), and `shared_ptr` is the obvious choice for a global variable that is passed around and copied in and out of functions.

Of course I might be wrong, and it would be nice to be able to test this change before merging to master :smile: (on the other hand the revert would be super simple)